packages/web-components/src/components/va-table/va-table-inner/va-table-inner.tsx

```diff
@@ -111,6 +111,61 @@ export class VaTableInner {
     }
   }
 
+  connectedCallback() {
+    this.setupEventDelegation();
+  }
+
+  setupEventDelegation() {
+    // Events to dispatch from interactive elements in the table cells
+    const dispatchedSlotEvents = [
+      'click', 'vaChange'
+    ];
```

Comment:
This approach would require us to add events that we'd like to dispatch up. In this example, I've added `click` and `vaChange` because those are the events that `va-button` and `va-checkbox` use.